src/appConstants/index.ts

```diff
@@ -4,3 +4,6 @@ export const AUTH_TOKEN = 'AUTH_TOKEN';
 export const SET_TOKEN = 'SET_TOKEN';
 
 export const BACKEND_LINK = 'https://rss-teams-dev.herokuapp.com/graphql';
```

Comment:
I'd recommend to foresee a possibility to switch between API servers for development and production.